csrc/preseg_passes/allocation_order_inference.cpp

```diff
@@ -40,6 +40,7 @@ class AllocationOrderInferencer : public IterVisitor {
  protected:
```

Comment:
Can you add a code comment that the ctor takes a partially populated alloc_order_map that contains the ground truth (I guess allocation order of inputs/outputs) and the inferencer tries to fill the rest? This bit wasn't immediately clear to me and caused some unnecessary confusion.